vcd/resource_vcd_catalog_media.go

```diff
@@ -62,7 +62,7 @@ func resourceVcdCatalogMedia() *schema.Resource {
 			},
 			"media_path": {
 				Type:        schema.TypeString,
-				Required:    true,
+				Optional:    true,
```

Comment:
The docs should still say that the `media_path` is required. The creation won't fail on schema validation, but it will fail within the Create function. So, for the user, nothing has changed, except the timing of failure.